skipper/cli.py

```diff
@@ -407,7 +407,8 @@ def _prepare_build_container(registry, image, tag, git_revision, container_conte
     else:
         build_context = '.'
 
-    command = ['build', '--network=host', '-t', tagged_image_name, '-f', docker_file, build_context]
+    command = ['build', '--network=host', '-t', tagged_image_name, '-f', docker_file, build_context,
+               "--ulimit", "nofile=65536:65536"]
```

Comment:
```suggestion
    ulimits = DEFAULT_SKIPPER_ULIMIT.split(',')  # format: x=y,a=b
    for limit in unlimits.split(','):
        command += ['--ulimit', limit]
```

Reply:
Consider adding a function in utils that returns a list with the ulimit for the command and call it from both places.